history.go

```diff
@@ -101,11 +102,21 @@ func (r *Recorder) Execute(command string) error {
 	return nil
 }
 
+// SetPath takes a string and set history log file path
+func (r *Recorder) SetPath(path string) {
```

Comment:
You could use functional options for this, couldn't you, as per Code Club?

Reply:
Yeah. Thank you!

Reply:
after second thought, I want to keep the variable private and have methods to deal with it. Is there anything wrong?